File: ION/packages/Nfinium/XFINIUM.PDF 5.4.0 Mobile Professional Edition Commercial Version/Samples/Common/BatesNumbers.cs
using System;
using System.IO;
using Xfinium.Pdf;
using Xfinium.Pdf.Graphics;
using Xfinium.Pdf.Content;

namespace Xfinium.Pdf.Samples
{
    /// <summary>
    /// BatesNumbers sample.
    /// </summary>
    public class BatesNumbers
    {
        /// <summary>
        /// Main method for running the sample.
        /// </summary>
        public static SampleOutputInfo[] Run(Stream input)
        {
            // Load the input file.
            PdfFixedDocument document = new PdfFixedDocument(input);

            PdfBatesNumberAppearance bna = new PdfBatesNumberAppearance();
            bna.Location = new PdfPoint(25, 5);
            bna.Brush = new PdfBrush(PdfRgbColor.DarkRed);

            PdfBatesNumberProvider bnp = new PdfBatesNumberProvider();
            bnp.Prefix = "XFINIUM";
            bnp.Suffix = "PDF";
            bnp.StartNumber = 1;

            PdfBatesNumber.WriteBatesNumber(document, bnp, bna);

            SampleOutputInfo[] output = new SampleOutputInfo[] { new SampleOutputInfo(document, "xfinium.pdf.sample.batesnumbers.pdf") };
            return output;
        }
    }
}

File: ION/packages/Nfinium/XFINIUM.PDF 5.4.0 Mobile Professional Edition Commercial Version/Samples/Common/BatesNumbers.vb
Imports System.IO
Imports Xfinium.Pdf
Imports Xfinium.Pdf.Graphics
Imports Xfinium.Pdf.Content

Namespace Xfinium.Pdf.Samples
	''' <summary>
	''' BatesNumbers sample.
	''' </summary>
	Public Class BatesNumbers
		''' <summary>
		''' Main method for running the sample.
		''' </summary>
		Public Shared Function Run(input As Stream) As SampleOutputInfo()
			' Load the input file.
			Dim document As New PdfFixedDocument(input)

			Dim bna As New PdfBatesNumberAppearance()
			bna.Location = New PdfPoint(25, 5)
			bna.Brush = New PdfBrush(PdfRgbColor.DarkRed)

			Dim bnp As New PdfBatesNumberProvider()
			bnp.Prefix = "XFINIUM"
			bnp.Suffix = "PDF"
			bnp.StartNumber = 1

			PdfBatesNumber.WriteBatesNumber(document, bnp, bna)

			Dim output As SampleOutputInfo() = New SampleOutputInfo() {New SampleOutputInfo(document, "xfinium.pdf.sample.batesnumbers.pdf")}
			Return output
		End Function
	End Class
End Namespace